velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -105,6 +105,99 @@ class PrestoSerializerTest : public ::testing::Test {
     return vectorMaker_->rowVector(childVectors);
   }
 
+  void writeInt32(OutputStream* out, int32_t value) {
+    out->write(reinterpret_cast<char*>(&value), sizeof(value));
+  }
+
+  void writeInt64(OutputStream* out, int64_t value) {
+    out->write(reinterpret_cast<char*>(&value), sizeof(value));
+  }
+
+  // Take a base vector and serialize it as an RLE vector
+  void serializeAsRle(
+      int32_t numValues,
+      RowVectorPtr rowValueVector,
+      std::ostream* output) {
+    // row must have a single vector
+    EXPECT_EQ(rowValueVector->size(), 1);
+
+    // value vector must have a single value
+    EXPECT_EQ(rowValueVector->childAt(0)->size(), 1);
+
+    const char codec = 0;
```

Comment:
This is a lot of copy-paste. Is there a way to avoid code duplication?

Reply:
I can extract the serialization part out of PrestoVectorSerializer as a static method + lambda writing actual data stream. 

However I'm not sure what would be the best way to expose it to both PrestoVectorSerializer and the PrestoSerializerTest as I'm total noob in C++ and Velox codebase/code practices.

```
void serializePage(
    int32_t numRows,
    OutputStream* out,
    std::function<void(OutputStream* out)> writeStreams) {
```